context/app/static/js/components/ServiceStatusTable/ServiceStatusTable.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import { StyledExternalLinkIcon } from 'js/components/files/GlobusLinkMessage/style';
+
+import { HeaderCell } from 'js/shared-styles/Table';
+import { LightBlueLink } from 'js/shared-styles/Links';
+
+import StatusIcon from './StatusIcon';
+import { useGatewayStatus } from './hooks';
+
+function buildServiceStatus(name, response, noteFunc) {
+  const { build, version, api_auth } = response;
+  const isUp = api_auth || name === 'gateway';
+  // The gateway isn't explicit: If it's not up, you wouldn't get anything at all,
+  // (and you wouldn't be able to get to the portal in the first place.)
+  return {
+    name,
+    github: build ? `https://github.com/hubmapconsortium/${name}` : undefined,
+    build,
+    version,
+    isUp,
+    note: noteFunc(response),
+  };
+}
+
+function ServiceStatusTable() {
+  const gatewayStatus = useGatewayStatus();
+
+  const apiStatuses = gatewayStatus
```

Comment:
This operation could be synced with changes to `gatewayStatus` so it doesn't run on each render, but I think it's lightweight enough that it doesn't matter.